src/test/java/cryptator/SolverTest.java

```diff
@@ -492,7 +489,7 @@ public void testSendMoreMoneyList3() throws CryptaParserException, CryptaModelEx
     public void testUnsatSendMoreMoneyList() throws CryptaParserException, CryptaModelException, CryptaSolverException {
         t.config.setHornerScheme(true);
         t.testUNSAT("send+more=money; s+e=n", "send+more=money;;; s+e=n", "send+more=money; s+e=n;",
-                "send+more=money&& s+e=n", "send+more=money&&&&&& s+e=n", "send+more=money&& s+e=n&&");
+                "send+more=money&& s+e=n", "send+more=money&&&&&& s+e=n");
```

Comment:
The cryptarithm `send+more=money&&&&&& s+e=n` should raise an exception.

However, `send+more=money ;; s+e=n ;` should not ?

Reply:
Ah, we don't accept multiple `&&` side by side, I missed it. I will correct this.